src/SyGuS.ml

```diff
@@ -83,9 +65,9 @@ let func_definition (f : func) : string =
   ^ ") " ^ (Type.to_string f.return) ^ " " ^ f.body ^ ")"
 
 let var_declaration ((var_name, var_type) : var) : string =
-  "(declare-var " ^ var_name ^ " " ^ (Type.to_string var_type) ^ ")"
+  "(declare-const " ^ var_name ^ " " ^ (Type.to_string var_type) ^ ")"
 
-let parse_sexps (sexps : Sexp.t list) : t =
+let parse_sexps (sexps : Sexp.t list) : t =  
```

Comment:
Can you enable `show trailing spaces` in your editor and remove all of them from all files?